### ISSUE TITLE
Update all "official" documentation

### ISSUE THREAD

**Issue by Nyholm** [mailgun/mailgun-php#387]
We need to update all PHP examples in https://github.com/mailgun/documentation before we can tag 3.0.0.

**Comment by gaurav-chandra** [comment]
+1

**Comment by Nyholm** [comment]
Will the maintainers allow a bunch of PRs? I would hate to make some PRs that never gets any attention.

**Comment by chekalskiy** [comment]
Looks like it will take a lot of time. Maybe you can release at least pre-release of 3rd version?

**Comment by Nyholm** [comment]
That sounds like an excellent idea. 
I've got a few more PRs that I want to merge, but we should aim for a pre-release within a few weeks.

**Comment by DavidGarciaCat** [comment]
Agree, some extra PRs and a few more testing to confirm everything works as expected could be good before the pre-release or the stable release

**Comment by 109digital** [comment]
Hello, any ETA for 3.0 and updated docs?

**Comment by DavidGarciaCat** [comment]
Hi @109digital 

We're working on it, and will try to do it as soon as possible. Sorry for the wait, I understand that it might be annoying.

Cheers,

**Comment by 109digital** [comment]
Thanks David.

Doesn't seem to have been any activity on here or in mailgun/documentation/issues/274 in a few months... there was talk of a pre-release back in January, is that still in the works?

**Comment by DavidGarciaCat** [comment]
Yes! We want to run a final suite of testing for the new code, remove any deprecated dependencies, and check a couple of additional things. We're aiming to release the new version soon.

**Comment by 109digital** [comment]
Hi @DavidGarciaCat , any progress on this? Thank you!

**Comment by DavidGarciaCat** [comment]
Hi @109digital 

I started an update on my local environment, but I didn't finish it due to I needed to take care of other priorities. Will work on it again as soon as possible.

Cheers,

**Comment by 109digital** [comment]
Thanks David.

So is the documentation in readme.md of the `master` branch still incorrect?

Should we instead follow the installation instructions and usage examples at: https://github.com/mailgun/mailgun-php/blob/2.x/README.md ?

**Comment by DavidGarciaCat** [comment]
This one should work; just change the method to get access to the helper that you need (instead of `messages()`: https://github.com/mailgun/mailgun-php#usage

Please note that if Mailgun API added new features then these ones might not be yet ready as part of the SDK

Cheers

**Comment by DavidGarciaCat** [comment]
Many docs were already updated, as per the last check I run on the Mailgun Documentation page. There's also another issue https://github.com/mailgun/mailgun-php/issues/756 that refers to some missing docs, so we can use that one to track the leftovers.
